CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Replace tachyons classes with design tokens.
 - Paddings and margins to match other components.
 - Update loading to use an overlay instead of content loader after initial mount.
+- New layout and behavior of search result filter 
```

Comment:
Why did you add this here? It should be under the `Unreleased` section

Reply:
You are right, my bad